DRAFT_RELEASE_NOTES.md

```diff
@@ -100,11 +104,16 @@ extension-pkg-whitelist = "pydantic"
 ignore-patterns = '.*pb2[\S]*.py'
 ```
 
+### Upgrade Steps for Transitioning from Orphedomos to Fabric8
```

Comment:
A: Maybe _"Upgrade Steps for Projects with Customized Orphedomos Configurations"_? Usually we try to convey what conditions would cause the instructions to be relevant to a project in these titles.